pkg/users/accounts-list.js

```diff
@@ -121,7 +121,7 @@ const getGroupRow = (group, accounts) => {
         },
         {
             title: group.members,
-            props: { width: 10, },
+            props: { width: 20, },
```

Comment:
Is this an independent fix, or should it be squashed into the main commit? At first sight, this change is responsible for the last "update pixel test" commit? Please squash these two then, and send it separately to get it out of the way.